compiler/safetylib/domains/safetyProduct.ml

```diff
@@ -578,11 +578,8 @@ module PIDynMake (PW : ProgWrap) : VDomWrap = struct
      must be uniquely characterized by their names. *)
   let ssa_main, pa_res =
     (* FIXME: code duplication! dirty hack *)
-    let is_move_op =
-      X86_arch_full.X86_core.aparams.ap_is_move_op
-    in
     let asmOp = Arch_extra.asm_opI X86_arch_full.X86_core.asm_e in
-    FSPa.fs_pa_make asmOp is_move_op PW.main
+    FSPa.fs_pa_make asmOp PW.main
 
   (* We compute the reflexive and transitive clojure of dp *)
```

Comment:
Not so frequent that I am able to find a typo in a comment (clojure -> closure) :-)

Reply:
Fixed!